rasa/nlu/classifiers/keyword_intent_classifier.py

```diff
@@ -155,7 +155,7 @@ def persist(self) -> None:
         with self._model_storage.write_to(self._resource) as model_dir:
             file_name = f"{self.__class__.__name__}.json"
             keyword_file = model_dir / file_name
-            utils.write_json_to_file(keyword_file.name, self.intent_keyword_map)
+            utils.write_json_to_file(str(keyword_file), self.intent_keyword_map)
```

Comment:
We were saving something outside of the `model_dir`. Then after fixing we still kept getting the empty resource from the cache 🤦